lib/android-tools-info.ts

```diff
@@ -159,6 +159,13 @@ export class AndroidToolsInfo implements IAndroidToolsInfo {
 
 	public validateJavacVersion(installedJavaVersion: string, options?: { showWarningsAsErrors: boolean }): IFuture<boolean> {
 		return ((): boolean => {
+			// Check for min supported Java version
+			let minJavaVersion = "1.8";
```

Comment:
is there any reason to add this code, several lines below there's a check for currently installed version and verification against AndroidToolsInfo.MIN_JAVA_VERSION
Can't you just set MIN_JAVA_VERSION to 1.8.0 and do not change anything in this method?